its/src/test/java/com/sonar/it/shared/ScannerCliTest.java

```diff
@@ -102,14 +102,15 @@ public void givenTestHtmlCode_whenScannerForCliIsUsed_doesNotLogCsharpWarning()
       .setProperty("sonar.cs.file.suffixes=", ".no_extension");
     BuildResult result = ORCHESTRATOR.executeBuild(scanner);
 
-    // https://github.com/SonarSource/sonar-dotnet/issues/6714
-    //assertThat(result.getLogsLines(l -> l.contains("WARN"))).isEmpty();
+    assertThat(result.getLogsLines(l -> l.contains("WARN"))).isEmpty();
     TestUtils.verifyNoGuiWarnings(ORCHESTRATOR, result);
   }
 
   private SonarScanner getSonarScanner(String projectKey, String projectDir) {
-    return SonarScanner.create(new File(projectDir))
+    File projectDirPath = new File(projectDir);
+    return SonarScanner.create(projectDirPath)
       .setProjectKey(projectKey)
+      .setProperty("sonar.projectBaseDir", projectDirPath.getAbsolutePath())
```

Comment:
@costin-zaharia-sonarsource I would like to get rid of the unrelated WARN messages in the logs - however I set the `sonar.projectBaseDir` and the WARN is still there - is this expected?

Reply:
We could though get rid of the warning by simulating the scanner behavior and populating `sonar.pullrequest.cache.basepath`.

Reply:
I've changed the comments as actually they are related to the usage of the Scanner CLI. Hopefully they are useful